### ISSUE TITLE
locale property in frontend config has no default

### ISSUE THREAD

**Issue by jakubklimek** [UnifiedViews/Core#344]
When I upgraded from 1.6.0 to 2.0.0, develop, frontend refuses to start due to missing locale in config.properties.

This should have defaulted to "en".

Probably related to: https://github.com/UnifiedViews/Core/pull/314
```
HTTP Status 500 - com.vaadin.server.ServiceException: cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.

type Exception report

message com.vaadin.server.ServiceException: cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.

description The server encountered an internal error (com.vaadin.server.ServiceException: cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.) that prevented it from fulfilling this request.

exception

javax.servlet.ServletException: com.vaadin.server.ServiceException: cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.
	com.vaadin.server.VaadinServlet.service(VaadinServlet.java:307)
	cz.cuni.mff.xrg.odcs.frontend.ODCSApplicationServlet.service(ODCSApplicationServlet.java:109)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
root cause

com.vaadin.server.ServiceException: cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.
	com.vaadin.server.VaadinService.handleExceptionDuringRequest(VaadinService.java:1464)
	com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1418)
	com.vaadin.server.VaadinServlet.service(VaadinServlet.java:305)
	cz.cuni.mff.xrg.odcs.frontend.ODCSApplicationServlet.service(ODCSApplicationServlet.java:109)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
root cause

cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: locale.
	cz.cuni.mff.xrg.odcs.commons.app.conf.AppConfig.getString(AppConfig.java:118)
	cz.cuni.mff.xrg.odcs.frontend.AppEntry.init(AppEntry.java:89)
	com.vaadin.ui.UI.doInit(UI.java:641)
	com.vaadin.server.communication.UIInitHandler.getBrowserDetailsUI(UIInitHandler.java:217)
	com.vaadin.server.communication.UIInitHandler.synchronizedHandleRequest(UIInitHandler.java:74)
	com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:41)
	com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1406)
	com.vaadin.server.VaadinServlet.service(VaadinServlet.java:305)
	cz.cuni.mff.xrg.odcs.frontend.ODCSApplicationServlet.service(ODCSApplicationServlet.java:109)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
note The full stack trace of the root cause is available in the Apache Tomcat/7.0.28 logs.

Apache Tomcat/7.0.28
```

**Comment by tomas-knap** [comment]
If locale is not available in config.properties, EN should be automatically used.

**Comment by tomas-knap** [comment]
Discussion in #303 should be finished first, it is a blocker for this issue

**Comment by mbl-eea-sk** [comment]
This is already solved in
https://github.com/UnifiedViews/Core/commit/fdf7d3a891d375a44cc0621bdc6e6eada10ecb96

**Comment by skrchnavy** [comment]
it is solved in v2.1.0
